docs/introduction.html

```diff
@@ -18,7 +18,7 @@
 <script><!--#include virtual="js/templateData.js" --></script>
 
 <script id="introduction-template" type="text/x-handlebars-template">
-  <h3> Kafka is <i>a distributed streaming platform</i>. What exactly does that mean?</h3>
+  <h3> Kafka&trade; is <i>a distributed streaming platform</i>. What exactly does that mean?</h3>
```

Comment:
I do not have a clear answer myself, since this is piggy-backed from the `kafka-site` diffs. cc @derrickdoo